pkg/config/config.go

```diff
@@ -519,8 +523,8 @@ func ConvertMapToCty(iMap map[string]interface{}) (map[string]cty.Value, error)
 // global variables, then they are replaced by the cty.Value of the
 // corresponding entry in the origin. All other cty.Values are unmodified.
 // ERROR: if (somehow) the cty.String cannot be converted to a Go string
-// ERROR: rely on HCL TraverseAbs to bubble up "diagnostics" when the global variable
-//        being resolved does not exist in b.Vars
+// ERROR: rely on HCL TraverseAbs to bubble up "diagnostics" when the global
+// variable being resolved does not exist in b.Vars
```

Comment:
I found this more clear before, though I'm not opposed to the change. Was this made by the go linter?

Reply:
A worse change change was proposed by the linter, and this is how I was able to appease the linter. 
I reverted it for now.